TAO/TAO_IDL/include/idl_global.h

```diff
@@ -426,6 +433,12 @@ class TAO_IDL_FE_Export IDL_GlobalData
   // BE calls to check whether pragma for this sequence has been set
   bool dcps_sequence_type_defined (const char* seq_type);
 
+  // FE calls when #pragma DCPS_DATA_MAP_TYPE is processed
+  void set_dcps_map_type (const char* map_type);
+
+  // BE calls to check whether pragma for this map has been set
+  bool dcps_map_type_defined (const char* map_type);
+
```

Comment:
I don't think this is needed.  There's some support for doing similar operations on sequences, but maps wouldn't be used that way.